source/guides/guides/command-line.md

```diff
@@ -303,7 +303,7 @@ cypress run --reporter junit --reporter-options mochaFile=result.xml,toConsole=t
 
 ### `cypress run --spec <spec>`
 
-Run tests specifying a single test file to run instead of all tests.
+Run tests specifying a single test file to run instead of all tests. The path should be relative to the current working directory.
```

Comment:
I thought this needs to be relevant to the project root? Not necessarily the cwd.

Reply:
it is relative to the current folder. For example, let's say we are outside the folder completely and use `--project` parameter. Notice how the `--spec` still uses relative to the CWD, not to the project:

<img width="1436" alt="Screen Shot 2021-01-29 at 8 57 39 AM" src="https://user-images.githubusercontent.com/2212006/106283528-0b173080-6210-11eb-8fd5-aeeff5541365.png">

We can also use the absolute path to spec

<img width="1432" alt="Screen Shot 2021-01-29 at 9 00 40 AM" src="https://user-images.githubusercontent.com/2212006/106283918-824cc480-6210-11eb-9993-9ef65a992afc.png">

But not the path relative to the project's root

<img width="1378" alt="Screen Shot 2021-01-29 at 9 00 10 AM" src="https://user-images.githubusercontent.com/2212006/106283945-8d9ff000-6210-11eb-88e2-6d08352e50f0.png">

Reply:
I have updated the file and added the clarification and more examples

Reply:
Oh, seems like the error we were going to throw always said it was relative to the project directory though.